src/pspm_init.m

```diff
@@ -257,8 +242,6 @@
 % Custom
 defaults.channeltypes(end+1) = struct(s_t, 'custom',           s_de, 'Custom',                                   s_i, @pspm_get_custom,    s_da, 'wave');
 
-defaults.findvalidfixations.channeltypes = {'gaze_pp_x',   'gaze_pp_y',   'gaze_x',   'gaze_y',   'pupil',   'pupil_missing',   'pupil_pp'}; % alphabetical order
```

Comment:
I do not remember if I wrote this or what this was used for. I checked the code and did not see this `findvaludfixations.channeltypes` is used by any function. I think it can be safely deleted now.